app/web/src/components/KeyValue/KeyValue.tsx

```diff
@@ -0,0 +1,57 @@
+import Svg from 'src/components/Svg/Svg'
+
+interface KeyValueType {
+  keyName: string
+  children: React.ReactNode
+  hide?: boolean
+  canEdit?: boolean
+  onEdit?: () => void
+  isEditable?: boolean
+  bottom?: boolean
+  className?: string
+}
+
+const KeyValue = ({
```

Comment:
Broke out into its own component to be used across both `PartProfile` and `UserProfile`

Reply:
I know you like this component but I kinda hate it, well hate is a sting word but the amount of input params is off putting for such a simple component. If we pulling it out to be reused I might be good to make a few changes.
- hide should be removed entirely, why did I added it? like should we add a hid param to every component that's conditionally rendered 😩 . Cases where it was being used should just be replaced with `{!shouldHide && <KeyValue ... />}
- `isEditable` is a terrible name, The variable is for when the component is in a edit state, but i reads like whether the component is allowed to be edited or something. perhaps 'inEditMode`
- I liked your variable name `hasPermissionToEdit` could we rename `canEdit` to that as well? much clearer.

How does that sound, would you mind?

Reply:
Yeah totally, love these ideas. I hear you it about the input params too.